Cubical/Categories/Displayed/Section.agda

```diff
@@ -102,6 +103,7 @@ module _ (C : Category ℓC ℓC') (P : Preorderᴰ C ℓD ℓD') where
     Section≡ _ (λ c → refl) (λ f → refl)
   PreorderSectionIsoCatSection .leftInv a = refl
 
+-- | TODO: this should go somewhere else
 module _ (C : Category ℓC ℓC') (D : Category ℓD ℓD') where
   weaken : Categoryᴰ C ℓD ℓD'
```

Comment:
why "weaken"

Reply:
The idea is to think of a displayed category `C` over a category `G` like a dependent type `G |- C`. Then we can think of an ordinary category as dependent on an empty context `* |- C` and so this operation takes `* |- D` and turns it into `C |- wk(D)`, i.e., weakening.